Data/_render_collections.qmd

```diff
@@ -371,13 +384,16 @@ for c in meta:
 
 {detailblock(collectionID,spatial,temporal,table,link_block,license_button,cite )}
 
+</div>
+
+</div>
 
 """
 
 ```
 
 ```{python}
-text = text + togglescript()
+text = Filter.createcheckboxes(list(set(categories))) + text + togglescript()
```

Comment:
```suggestion
text = Filter.createcheckboxes(sorted(set(categories))) + text + togglescript()
```

Converting the set to list is not necessary, Python can iterate over sets just fine. However, you probably want your categorties to be sorted alphabetically